clients/admin-ui/src/features/system/system-form-declaration-tab/PrivacyDeclarationFormTab.tsx

```diff
@@ -264,7 +264,7 @@ const PrivacyDeclarationFormTab = ({
           dictAvailable={features.dictionaryService}
           handleAdd={handleOpenNewForm}
           handleDictSuggestion={handleOpenDictModal}
-          vendorSelected={!!system.meta.vendor}
+          vendorSelected={!!system.meta?.vendor}
```

Comment:
@Kelsey-Ethyca should this be switched to the new `system.vendor` field?

Reply:
oh good catch! looks like that one and also this need to be updated:
https://github.com/ethyca/fides/blob/bd9d6070196451c830e48960433c1e0bee38208b/clients/admin-ui/src/features/system/system-form-declaration-tab/PrivacyDeclarationFormTab.tsx#L317